Add, remove and update icons

alex-page opened the pull request:
### WHY are these changes introduced?

Brings in the new SVG files from https://github.com/Shopify/polaris/issues/11088.

- [x] Only uses the major icons and does not include minor icons
- [x] Runs svgo to optimize the svg files
- [x] Removes `fill="none"` from `<svg>` tags
- [x] Removes `clip-rule="` on `<path>` tags
- [x] Replaces #4A4A4A with #5C5F62 for the fill color of icons

### WHAT is this pull request doing?

Update icons for `Jobs`, `JobsFilled`, `Automation`, `AutomationFilled`, `GiftCard`, `InventoryFilled`, `Reports`, `ShipmentFilled`, `AppsFilled`, `Markets`, `MarketsFilled`, `BillingStatementDollarFilled`, `BillingStatementEuroFilled`, `BillingStatementPoundFilled`, `BillingStatementRupeeFilled`, `BillingStatementYenFilled`, `LegalFilled`, `IdentityCard`, `IdentityCardFilled`, `Domains`, `DomainNew`, `DomainsFilled`, `Store`, `OnlineStore`, `ManagedStore`, `ImportStore`, `Tax`, `TaxFilled`, `PackageFilled`, `Location`, `LocationFilled`, `InactiveLocation`, `Pin`, `Globe`, `Transport`, `Nature`, `Food`, `Colors` to use the new design language

Added new icons for `ShippingLabel`, `ShippingLabelFilled`, `Bill`, `BillFilled`, `UsersPermissions`, `UsersPermissionsFilled`, `ActivityLog`, `ActivityLogFilled`, `Instagram`, `Tips`

Removed icons `ReportFilledMinor`, `ReportMinor`, `ListMajor`, `ListFilledMajor`

alex-page committed:
Add, remove and update icons

alex-page committed:
Remove clip-rule on <path>, remove fill none on <svg>, replace fill #4A4A4A with #5C5F62

alex-page committed:
Re run svgo after cleaning up svgs

sam-b-rose commented on polaris-icons/icons/ActivityLog.yml:
```diff
@@ -0,0 +1,12 @@
+  - Activity
+authors:
+  - José Torre
+version: 1
```
If we do chose to remove the version (#11127), we should remember to update these here as well 👍

sam-b-rose committed:
Make changelog entries past tense

zecarlostorre commented on polaris-icons/icons/ActivityLog.yml:
```diff
@@ -0,0 +1,12 @@
+name: Activity log
+description: N/A
+keywords:
```
History
Timeline
Events
Past
Monitoring

zecarlostorre commented on polaris-icons/icons/ActivityLogFilled.yml:
```diff
@@ -0,0 +1,12 @@
+name: Activity log filled
+description: N/A
+keywords:
```
History
Timeline
Events
Past
Monitoring

zecarlostorre commented on polaris-icons/icons/ActivityLog.yml:
```diff
@@ -0,0 +1,12 @@
+  - Page
+  - Log
+  - Activity
+authors:
```
+ Holly Schofield

zecarlostorre commented on polaris-icons/icons/ActivityLogFilled.yml:
```diff
@@ -0,0 +1,12 @@
+  - Log
+  - Activity
+authors:
+  - José Torre
```
+ Holly Schofield

zecarlostorre commented on polaris-icons/icons/AppsFilled.yml:
```diff
@@ -9,7 +9,8 @@ keywords:
   - extensions
 authors:
   - Joe Thomas
+  - José Torre
```
+ Holly Schofield

zecarlostorre commented on polaris-icons/icons/ActivityLog.yml:
```diff
@@ -0,0 +1,12 @@
+name: Activity log
+description: N/A
```
Used to represent a story activity log, or any other report/series of events that happened in the past.

zecarlostorre commented on polaris-icons/icons/ActivityLogFilled.yml:
```diff
@@ -0,0 +1,12 @@
+name: Activity log filled
+description: N/A
```
Used to represent a story activity log, or any other report/series of events that happened in the past.

zecarlostorre commented on polaris-icons/icons/Bill.yml:
```diff
@@ -0,0 +1,10 @@
+name: Bill
+description: N/A
```
User to represent Bill Pay, or a receipt.

zecarlostorre commented on polaris-icons/icons/BillFilled.yml:
```diff
@@ -0,0 +1,10 @@
+name: Bill filled
+description: N/A
```
User to represent Bill Pay, or a receipt.

zecarlostorre commented on polaris-icons/icons/Bill.yml:
```diff
@@ -0,0 +1,10 @@
+name: Bill
+description: N/A
+keywords:
+  - N/A
```
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

zecarlostorre commented on polaris-icons/icons/BillFilled.yml:
```diff
@@ -0,0 +1,10 @@
+name: Bill filled
+description: N/A
+keywords:
```
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

zecarlostorre commented on polaris-icons/icons/BillingStatementDollarFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
- Dollar
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

zecarlostorre commented on polaris-icons/icons/BillingStatementEuroFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
- Euro
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

zecarlostorre commented on polaris-icons/icons/BillingStatementPoundFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
- Pound
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

zecarlostorre commented on polaris-icons/icons/BillingStatementRupeeFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
- Rupee
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

zecarlostorre commented on polaris-icons/icons/BillingStatementYenFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
- Yen
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

zecarlostorre commented on polaris-icons/icons/BillingStatementDollarFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/BillingStatementEuroFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/BillingStatementPoundFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/BillingStatementRupeeFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/BillingStatementYenFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/ColorNone.yml:
```diff
@@ -14,7 +14,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/DomainNew.yml:
```diff
@@ -3,9 +3,9 @@ description: N/A
 authors:
-  - N/A
+  - José Torre
   - Joe Thomas
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/DomainNew.yml:
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
```
- Register
- Add
- Domain
- Web
- Create
- Website
- Digital

zecarlostorre commented on polaris-icons/icons/DomainRedirect.yml:
```diff
@@ -6,7 +6,8 @@ keywords:
 authors:
   - Kyryll Odobetskiy
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/Domains.yml:
```diff
@@ -3,9 +3,9 @@ description: N/A
   - N/A
 authors:
-  - N/A
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/Domains.yml:
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Domain
Web
Website
Digital

zecarlostorre commented on polaris-icons/icons/DomainsFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - discount
```
Remove:
- Discount

Add:
- Domain
- Web
- Website
- Digital

zecarlostorre commented on polaris-icons/icons/DomainsFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - discount
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/GiftCard.yml:
```diff
@@ -7,7 +7,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/GiftCardFilled.yml:
```diff
@@ -6,7 +6,8 @@ keywords:
   - present
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/Globe.yml:
```diff
@@ -5,7 +5,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/ImportStore.yml:
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Import
Store

zecarlostorre commented on polaris-icons/icons/InactiveLocation.yml:
```diff
@@ -8,7 +8,8 @@ authors:
   - Shanique Shields
   - James Findlater
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/Jobs.yml:
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Job
Briefcase
Work
Luggage
Business

zecarlostorre commented on polaris-icons/icons/JobsFilled.yml:
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
Job
Briefcase
Work
Luggage
Business

zecarlostorre commented on polaris-icons/icons/Location.yml:
```diff
@@ -6,7 +6,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/LocationFilled.yml:
```diff
@@ -5,7 +5,8 @@ keywords:
   - pin
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/Markets.yml:
```diff
@@ -8,7 +8,8 @@ keywords:
 authors:
   - Justina Chua
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/MarketsFilled.yml:
```diff
@@ -7,7 +7,8 @@ keywords:
   - global
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/Nature.yml:
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Nature
Leaf

zecarlostorre commented on polaris-icons/icons/PackageFilled.yml:
```diff
@@ -7,7 +7,8 @@ keywords:
   - cardboard
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/PasskeyFilled.yml:
```diff
@@ -6,7 +6,8 @@ keywords:
   - security
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/ProductsFilled.yml:
```diff
@@ -6,7 +6,8 @@ keywords:
   - price
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/Reports.yml:
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
 authors:
-  - N/A
```
Miru Alves

zecarlostorre commented on polaris-icons/icons/Reports.yml:
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Report
Explore

zecarlostorre commented on polaris-icons/icons/ShippingLabel.yml:
```diff
@@ -0,0 +1,10 @@
+name: Shipping label
+description: N/A
```
Used to represent shipping labels.

zecarlostorre commented on polaris-icons/icons/ShippingLabel.yml:
```diff
@@ -0,0 +1,10 @@
+name: Shipping label
+description: N/A
+keywords:
+  - N/A
```
Label
Shipping
Delivery
Package
Postage
Parcel

zecarlostorre commented on polaris-icons/icons/ShippingLabelFilled.yml:
```diff
@@ -0,0 +1,10 @@
+name: Shipping label filled
+description: N/A
+keywords:
+  - N/A
```
Label
Shipping
Delivery
Package
Postage
Parcel

zecarlostorre commented on polaris-icons/icons/ShippingLabelFilled.yml:
```diff
@@ -0,0 +1,10 @@
+name: Shipping label filled
+description: N/A
```
Used to represent shipping labels.

zecarlostorre commented on polaris-icons/icons/ShippingLabelFilled.yml:
```diff
@@ -0,0 +1,10 @@
+keywords:
+  - N/A
+authors:
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/ShippingLabel.yml:
```diff
@@ -0,0 +1,10 @@
+keywords:
+  - N/A
+authors:
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/Tax.yml:
```diff
@@ -7,7 +7,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/TaxFilled.yml:
```diff
@@ -6,7 +6,8 @@ keywords:
   - tax
 authors:
   - Joe Thomas
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/UsersAndPermissions.yml:
```diff
@@ -0,0 +1,10 @@
+name: Users and permissions
+description: N/A
```
Used to represent access permissions for users.

zecarlostorre commented on polaris-icons/icons/UsersAndPermissionsFilled.yml:
```diff
@@ -0,0 +1,10 @@
+name: Users and permissions filled
+description: N/A
```
Used to represent access permissions for users.

zecarlostorre commented on polaris-icons/icons/UsersAndPermissions.yml:
```diff
@@ -0,0 +1,10 @@
+name: Users and permissions
+description: N/A
+keywords:
+  - N/A
```
Lock
Person
Access
Control
Rights
Roles
Security
Authentication
Authorization
Login
Password

zecarlostorre commented on polaris-icons/icons/UsersAndPermissionsFilled.yml:
```diff
@@ -0,0 +1,10 @@
+name: Users and permissions filled
+description: N/A
+keywords:
+  - N/A
```
Lock
Person
Access
Control
Rights
Roles
Security
Authentication
Authorization
Login
Password

zecarlostorre commented on polaris-icons/icons/UsersAndPermissionsFilled.yml:
```diff
@@ -0,0 +1,10 @@
+keywords:
+  - N/A
+authors:
+  - José Torre
```
Holly Schofield

zecarlostorre commented on polaris-icons/icons/UsersAndPermissions.yml:
```diff
@@ -0,0 +1,10 @@
+keywords:
+  - N/A
+authors:
+  - José Torre
```
Holly Schofield

zecarlostorre reviewed (approved):
The icons look good, so I'm approving this.
If you want to include it, I added Holly's name to the icons she should be added to.
I also added descriptions and keywords to icons that didn't have them.

alex-page committed:
Remove version from icons yml file (#11127)

### WHY are these changes introduced?

The version value in the icons yml file is confusing and duplicated with
the package version. This removes the extra version value in each icon
file to only use the package version.

### WHAT is this pull request doing?

Removes the confusing version value in the icons `.yml` files.
